catroid/src/androidTest/java/org/catrobat/catroid/test/io/asynctask/ProjectUnzipAndImportTaskTest.java

```diff
@@ -52,9 +52,9 @@
 @RunWith(AndroidJUnit4.class)
 public class ProjectUnzipAndImportTaskTest {
 
-	private static final String AIR_FIGHT_0_5 = "Air fight 0.5";
-	private static final String FALLING_BALLS = "Falling balls";
-	private static final String AIR_FIGHT_0_5_1 = "Air fight 0.5 (1)";
+	private static final String AIR_FIGHT_0_5 = "Air fight 0.5 (1)";
```

Comment:
This leads to a failure in line 89 further below.

Stacktrace:
https://jenkins.catrob.at/job/Catroid/job/PR-3973/51/testReport/org.catrobat.catroid.test.io.asynctask/ProjectUnzipAndImportTaskTest/All___1___Instrumented_Unit_Tests___testUnzipAndImportSingleProject/

=>
```
java.lang.AssertionError:
Expected: a collection containing "Air fight 0.5 (1)"
but: was "Air fight 0.5"
at org.catrobat.catroid.test.io.asynctask.ProjectUnzipAndImportTaskTest.testUnzipAndImportSingleProject(ProjectUnzipAndImportTaskTest.java:89)
```